docs/docs/what-about-generators.md

```diff
@@ -2,3 +2,60 @@
 id: what-about-generators
 title: What About Generators?
 ---
+
+## Promises compliment generators and coroutines!
+
+A lot of people tend to think that generators/coroutines and promises are two different ways of modelling asynchronous behavior.
+
+They're not!
+
+With the use of `Promise.coroutine()` we can turn any `GeneratorFunction` constructor (any generator) into a coroutine
```

Comment:
Prefer passive instead of first person. E.g. "With the use of `Promise.coroutine()` any `GeneratorFunction` constructor (any generator) can be turned into a coroutine"